app/views/admin/druid_search/new.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="row align-items-start">
       <%= render Elements::Forms::LabelComponent.new(form:, field_name: :druid, label_text: 'Search for DRUID', classes: 'col-md-2') %>
       <div class="col-md-6">
-        <%= form.text_field :druid, class: 'form-control', aria: Elements::Forms::InvalidFeedbackSupport.arias_for(form:, field_name: :druid) %>
+        <%= form.text_field :druid, class: 'form-control', aria: invalid_feedback_arias_for(form:, field_name: :druid) %>
```

Comment:
Ah, nevermind! This isn't going to work here because this is a view, not a component. 😆  Walking away from the spike.

Reply:
And that is what led me to extract it in the first place ...